docs/sharding.md

```diff
@@ -0,0 +1,42 @@
+# Multiple Azure SignalR Service Instances Support
+Currently we support multiple Azure SignalR Service instances **for Persistent mode only**.
+
+## Set up multiple endpoints
+To enable multiple endpoints support, you should: 
+
+1. Set the `ServiceTransportType` to `Persistent` in your configuration.
+```json
+{
+    "AzureSignalRServiceTransportType":"Persistent"
+}
+```
+
+2. Set multiple SignalR connection string entries in your configuration. The keys are in the following format : 
+```
+AzureSignalRConnectionString: <name> : <role>
```

Comment:
<ConnectionStringSetting>: <name> : <role>
* ConnectionStringSetting: describe it
* ...

Reply:
Now `ConnectionStringSetting` is not applicable to service endpoints.